### PR TITLE
Add kanemorgan and tomwhite

### PR DESCRIPTION
**Pull request by KaneMorgan**

### CODE REVIEW

File: modules/users/manifests/kanemorgan.pp [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class users::kanemorgan {
+  govuk_user { 'kanemorgan':
+    fullname => 'Kane Morgan',
+    email    => 'kane.morgan@digital.cabinet-office.gov.uk',
+    ssh_key  => [
+    'ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAtVh8j/aL8uUe5F9WKMw/qCK/KgWeQk+JC5lkSNkTgFrSi5cG1zAEpoGimfvynWZQrVbe4YnC9OQ5yQtubKDcj52ZV0D6TL1jbMZ5ef0GTsPJMxIYtWBjvUlsHwKfQsV2rSYQHKX3ZcvjZDifMlB6ocxp7g8zbQoliWkPG5zTa1ltPDrMLzYOLw468MoqsE7bytLhLjUumDgkfimxVtQoVJQlnJRiIZmqegpq8F69B91tafi5hP75KbbrbDOB/LYsbnq2Gh3jBZKU/mrr807vNAgyYyhxx7QozrL4lEAELIV5SnmYrXGIQBdrD0+rTN0W7hLzqbyqdJ7pjQCYHT1w6w== kane.morgan@digital.cabinet-office.gov.uk'
+    ]
```
Comment (surminus) [inline modules/users/manifests/kanemorgan.pp]: This doesn't need to be an array if it's just a single key

File: modules/users/manifests/tomwhite.pp [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class users::tomwhite {
+  govuk_user { 'tomwhite':
+    fullname => 'Tom White',
+    email    => 'tom.white@digital.cabinet-office.gov.uk',
+    ssh_key  => [
+     'ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDRwl36fTtNjOEzYJ1jvSMM8H3wGO9ZMKrFEkA8QfV+G4kTKSRTt2hS55GY/7i24hd5SDak/pOdGDl9SdjV7RoYSYvnC+QeciA0CKybi0yo38ofJ53iaWSgf71GhoIsA2B7rUMcjPqq8joVnxy78hliv0HZf7mXC449Nb2TDylMEReyR8cY4gxObLVb9x4XmD2D4mANmUkN/g6KUTlaG/Z2nkZlWWY3NgZUHOpv+iAzV7lDmdOsHMq4OTGJ4mk1J2ksOZfO1PhqGlphEx2Bk030kzg82bjfIRSXeQwpxQcDwLgY+Y/XFuP3MnbUQZOD2lcPK8HWZngOSwH2hD0PojNV tom.white@MD031.local.'
+    ]
```
Comment (surminus) [inline modules/users/manifests/tomwhite.pp]: This doesn't need to be an array if it's just a single key

File: hieradata/integration.yaml [hunk 3]
```diff
@@ -322,3 +324,5 @@ users::usernames:
   - tijmenbrommet
   - timblair
   - tomhipkin
+  - kanemorgan
+  - tomwhite
```
Comment (surminus) [inline hieradata/integration.yaml]: Please sort these into alphabetical order :)

**Comment by surminus** [comment]
This is complaining because the keys need to 4096 bits to pass our tests.

**Comment by alexmuller** [comment]
Tests have passed on Travis. They won't run on Jenkins because this is from a fork of the repo :+1:

**Pull request merged**